scripts/init_env.sh

```diff
@@ -4,6 +4,7 @@ EVE_PYTHON_CMD=${EVE_PYTHON_CMD:-python3}
 EVE_ROOT=$(dirname $(dirname "$(readlink -e $0)"))
 
 MAKE_VENV="true"
+DEV_INSTALL="true"
```

Comment:
@egparedes What do you think about this change?
@tehrengruber Alternative solution could be just pip installing dependencies in the straight-forward way for CI.

Reply:
I'd go with #57. Will also test if actually the dependencies are correct once the TODO is resolved.